src/server/survey_and_program_data.R

```diff
@@ -151,16 +151,13 @@ surveyAndProgramData <- function(input, output, state, spectrumFilesState) {
         }"
 
     output$hot_survey <- rhandsontable::renderRHandsontable({
-        rhandsontable::rhandsontable(state$survey_data_human_readable(),
-        colHeaders = c("Country or region","Survey Id","Year","Age Group","Sex","HIV Status","Estimate (%)", "Standard Error (%)",
```

Comment:
This `colHeaders` property is order-sensitive, so this was actually a bug - if users uploaded a CSV with a different ordering the model would work but the UI would be wrong!